python/pyspark/sql/functions.py

```diff
@@ -2448,6 +2448,8 @@ def udf(f=None, returnType=StringType()):
 @since(2.3)
 def pandas_udf(f=None, returnType=None, functionType=None):
     """
+    .. note:: Experimental
+
```

Comment:
@BryanCutler, can we add a short node for createDataFrame and to Pandas with Pandas DataFrame input/output?

Reply:
Yeah, I could have sworn that was added before but I don't see it anywhere.  Would it make sense to add the note to the SQLconf description?

Reply:
Anywhere is fine IMHO if it looks making sense. Another option is  things like: 

```
.. versionchanged:: 2.3
  Arrow optimization added
  .. note:: Experimental. 
```

I just wanted to make sure this feature is experimental. If it's wrongly designed, we should consider a behaviour change as the last resort.

Reply:
BTW, don't forget to update `group.apply()`.

Reply:
Trying to add the note under `versionchanged` ends up with everything on one line and the note isn't highlighted like usual with Experimental, so I just made it it's own note.